### ISSUE TITLE
Finalize scBasset

### ISSUE THREAD

**Issue by adamgayoso** [scverse/scvi-tools#1840]
- [ ] Verify that all parameters are being initialized equivalently, alter the ones used here if needed
- [ ] Add more content to the tutorial that reproduces their results, allowing us to have some form of reproducibility
- [ ] Complete the user guide

**Comment by martinkim0** [comment]
@adamgayoso Thoughts on adding a batch correction portion to the tutorial using the buenrostro dataset from the original manuscript?

**Comment by adamgayoso** [comment]
I think we should and the maybe use scib metrics on it

**Comment by martinkim0** [comment]
I'm wondering whether we should add an option in `setup_anndata` to indicate whether the data is transposed already or not, where if it's not transposed, we do it on our end. It might be confusing for users when you have to `bdata = adata.transpose()` but then the latent space is saved back into the original `adata`.

**Comment by adamgayoso** [comment]
I agree this needs improvement. I'm a fan of what you see is what you get and it would be confusing if `model.adata` was transposed
